index.html

```diff
@@ -3,17 +3,93 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,700&display=swap" rel="stylesheet">
   <title>My Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
+  
```

Comment:
I can see you forgot to add the viewport meta tag. Without the tag, your media queries won't work